evmbin/src/display/json.rs

```diff
@@ -93,26 +141,30 @@ impl vm::Informant for Informant {
 					println!("{}", trace);
 				}
 
-				let success_msg = json!({
-					"output": format!("0x{}", success.output.to_hex()),
-					"gasUsed": format!("{:#x}", success.gas_used),
-					"time": display::as_micros(&success.time),
-				});
+				let message_success =
+					MessageSuccess {
+						output: &format!("0x{}", success.output.to_hex()),
```

Comment:
```suggestion
						output: &format!("{:#x}", success.output.to_hex()),
```

Reply:
I think you mean to skip `.to_hex()` too, no?

Reply:
I've been trying to get this to compile for a while with a few different approaches. I've pushed the different approaches together in this commit https://github.com/paritytech/parity-ethereum/commit/5fbea495c8adcb1cb8de08abdbcb1c13d0e2c0d3#diff-98c535329ae4c9199c5b7ddd6b06b5ba of separate branch 'luke-9459-evmbin-serde-hex-format'.

i've been trying to "implement LowerHex for Vec<u8>" but when i build with `cargo build -p evmbin --release` i still get errors like the following that I haven't been able to overcome:
```
106 |             write!(f, "{:x}", val)
    |                               ^^^ the trait `std::fmt::LowerHex` is not implemented for `std::vec::Vec<u8>`

199 |                         output: &format!("{:#x}", success.output),        
    |                                                   ^^^^^^^^^^^^^^ the trait `std::fmt::LowerHex` is not implemented for `std::vec::Vec<u8>`
```

Reply:
Ok, it's not a big deal I think, thank you for trying! Apparently `LowerHex` for byte slices (or vectors) have a long history of discussion and isn't likely to happen soon.